tests/jballerina-unit-test/src/test/resources/test-src/expressions/invocations/function-invocation-expr.bal

```diff
@@ -354,6 +354,46 @@ function testTypeRefTypedRestArg() {
 function testFn(int a, string... b) returns int =>
     b.reduce(function (int tot, string str) returns int => tot + str.length(), a);
 
+function testFuncWithNilReturnTypeWithoutVariableAssignment() {
+    stream<string> testStream = new;
+    testStream.close(); // No error
+    f1(); // No error
+    f2(); // No error
+    f4(); // No error
+    f5(); // No error
+    f6(); // No error
```

Comment:
I don't think we need the "No error" comments here since they are applicable to all and are in a positive test.